client.go

```diff
@@ -162,12 +172,32 @@ var h2Transport = http2.Transport{
 // NewClient creates a RESTful client instance.
 // The instance has a semi-permanent transport TCP connection.
 func NewClient() *Client {
+	return NewClientWInterface("")
+}
+
+// NewClientWInterface creates a RESTful client instance with binded to that interface.
```

Comment:
"...client instance with binded to that interface." -> "...client instance bound to that network interface."